### ISSUE TITLE
Not connecting to wifi

### ISSUE THREAD

**Issue by sasanbozorg** [MatsuriDayo/NekoBoxForAndroid#292]
Hello.. the configurations in the nekobox app do not work with wifi. There is no problem with SIM card internet.The same configurations in the v2rayng app do not have a problem with wifi.
Please help if anyone has such a problem

**Comment by janahnovin** [comment]
Yes, I have exactly the same problem. I think this is a bug that should be fixed

**Comment by sasanbozorg** [comment]
> Yes, I have exactly the same problem. I think this is a bug that should be fixed

to solve the problem
go to settings - dsn settings - replace remote dns to 8.8.8.8 and direct dns 223.5.5.5
